ExampleApp/Base.lproj/Main.storyboard

```diff
@@ -72,5 +72,53 @@
             </objects>
             <point key="canvasLocation" x="-678" y="-290"/>
         </scene>
+        <!--Example View Controller-->
+        <scene sceneID="uFO-j0-SXY">
```

Comment:
Why is this VC new?

Reply:
To fix some silliness that occurs when reusing the storyboard nib for advanced options screen.